inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -28,22 +38,25 @@ class WindTunnel(Scenario):
     """
 
     def __init__(self,
-                 flow_velocity: List[float],
-                 domain: Optional[List[float]] = None):
+                 flow_velocity: List[float] = None,
+                 domain: Optional[dict] = None):
         """Initializes the `WindTunnel` conditions.
         
         Args:
-            flow_velocity: Velocity of the air flow in m/s.
-            domain: List containing the lower and upper boundary of the wind
-                tunnel in each (x, y, z) direction. It is the natural
-                description with the default OpenFOAM simulator.
+            flow_velocity (dict): Velocity of the air flow in m/s.
+            domain (dict): List containing the lower and upper boundary of
+                the wind tunnel in each (x, y, z) direction. It is the
+                natural description with the default OpenFOAM simulator.
         """
-
-        self.flow_velocity = flow_velocity
+        if flow_velocity is None:
+            logging.info("Using a default flow velocity: [30, 0, 0].")
+            self.flow_velocity = [30, 0, 0]
+        else:
+            self.flow_velocity = flow_velocity
 
         if domain is None:
             logging.info("Using a default domain: [[-5, 15], [-4, 4], [0, 8]].")
-            self.domain = [[-5, 15], [-4, 4], [0, 8]]
+            self.domain = {"x": [-5, 15], "y": [-4, 4], "z": [0, 8]}
```

Comment:
This seems to be even more complex, since simulation users are not so familiar with initialising classes to add domain data.

To be honest, it seems very intuitive to be a list and a dict or a dataclass needs to be instructed to the user. At this point, I will just add a condition to verify if it is a dict.